### PR TITLE
[feat] Added Text utilities and List Comp

### PR DESCRIPTION
**Pull request by rishichourasia**
Files to Check -  

typography.css
typography.html

list.css
list.html

### Live Preview for added components to check - 
https://deploy-preview-16--scale-ui.netlify.app/components/typography/typography

https://deploy-preview-16--scale-ui.netlify.app/components/list/list

### CODE REVIEW

File: Components/List/list.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	list-style: none;
+}
+.card-title {
+	font-size: 1.2rem;
+	font-weight: 600;
+}
+.sub-text {
+	font-size: 1rem;
+	font-weight: 400;
+	color: grey;
```
Comment (sakshi006) [inline Components/List/list.css:66]: use CSS variables for colors

File: Components/Typography/typography.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+}
+.is-right {
+	text-align: right;
+}
+.is-centre {
+	text-align: center;
+}
+
+.is-dark {
+	color: black;
```
Comment (sakshi006) [inline Components/Typography/typography.css:57]: Here as well

**Review by sakshi006 — APPROVED** [review]
Looks good to merge

File: Components/List/list.html [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
...-list%2522%253E%250A%2509%2509%2509%253Cli%2520class%253D%2522list-items%2522%253E%250A%2509%2509%2509%2509%253Cdiv%2520class%253D%2522card-horizontal%2522%253E%250A%2509%2509%2509%2509%2509%253Cdiv%2520class%253D%2522card-img%2522%253E%250A%2509%2509%2509%2509%2509%2509%253Cimg%2520src%253D%2522%252Fassets%252Fimages%252Fluca-bravo-8Nql6oVT_1A-unsplash.jpg%2522%2520alt%253D%2522card-img%2522%252F%253E%250A%2509%2509%2509%2509%2509%253C%252Fdiv%253E%250A%2509%2509%2509%2509%2509%253Cdiv%2520class%253D%2522card-contents%2522%253E%250A%2509%2509%2509%2509%2509%2509%253Cdiv%2520class%253D%2522card-title%2522%253E%250A%2509%2509%2509%2509%2509%2509%2509Card%2520Title%250A%2509%2509%2509%2509%2509%2509%2509%253Cp%2520class%253D%2522sub-text%2522%253ESub-text%253C%252Fp%253E%250A%2509%2509%2509%2509%2509%2509%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2509%253Cdiv%2520class%253D%2522card-description%2522%253ELorem%2520Ipsem%253C%252Fdiv%253E%250A%2509%2509%2509%2509%2509%253C%252Fdiv%253E%250A%2509%2509%2509%2509%253C%252Fdiv%253E%250A%2509%2509%2509%253C%252Fli%253E%250A%2509%2509%253C%252Ful%253E"
```
Comment (sashaboi) [inline Components/List/list.html:188]: Could the carbon code snippet be modified in such a way that the list feature is show more prominently? as currently it just shows one card in a ul , the point of this list feature is to show multiple.

**Review by sashaboi — APPROVED** [review]
Looks good to go, do have a look at suggestion made for carbon text.

**Pull request merged**